docs/admin_guide.md

```diff
@@ -132,7 +133,6 @@ Other important `.env` configs:
 KBIN_STORAGE_URL=https://domain.tld/media
 
 # Ubuntu 22.04 installs PostgreSQL v14 by default, Debian 12 PostgreSQL v15 is the default
-POSTGRES_VERSION=14
```

Comment:
You removed `POSTGRES_VERSION=14`??? Maybe you need some sleep 😪  ?

Reply:
Nah it's a bit early for that yet